keras/backend/theano_backend.py

```diff
@@ -584,6 +584,11 @@ def softplus(x):
     return T.nnet.softplus(x)
 
 
+def softsign(x):
+    # return T.nnet.softmax(x)
```

Comment:
Well spotted, but given that a function called **abs** is defined elsewhere in the file, I thought it would be ok.

I tested it and it is still giving me ```loss: nan```